src/avrdude.conf.in

```diff
@@ -1479,6 +1479,24 @@ programmer # serialupdi
     hvupdi_support         = 1;
 ;
 
+#------------------------------------------------------------
+# serprog
+#------------------------------------------------------------
+
+# The Serprog programmer implementation allows programming
+# with programmers using the serprog protocol from flashrom.
+# See https://flashrom.org/supported_hw/supported_prog/serprog/overview.html
+# and https://flashrom.org/supported_hw/supported_prog/serprog/serprog-protocol.html
+# for details.
+
+programmer # serprog
+    id                     = "serprog";
+    desc                   = "Serprog";
```

Comment:
desc should be a one-line description (check how other programmer entries set this for style and scope). `desc` is printed on the `-c?` option:
```
$ avrdude -c? -pm328p |& grep -C5 serprog | sed "s/ via ISP//"
  pony-stk200       = Pony Prog STK200
  ponyser           = design ponyprog serial, reset=!txd sck=rts sdo=dtr sdi=cts
  powerdebugger_dw  = Atmel PowerDebugger (ARM/AVR) via debugWIRE
  powerdebugger_isp = Atmel PowerDebugger (ARM/AVR)
  raspberry_pi_gpio = Bitbang Raspberry Pi GPIO via linuxgpio (sysfs or libgpiod)
  serprog           = Serprog
  siprog            = Lancos SI-Prog (same as ponyser)
  snap_isp          = MPLAB(R) SNAP
  sp12              = Steve Bolt's Programmer
  stk200            = STK200
  stk500            = Atmel STK500 (probes STK500v2 first then STK500v1)
```